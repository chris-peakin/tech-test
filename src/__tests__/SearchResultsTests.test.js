import renderer from "react-test-renderer";
import SearchResults from "../components/SearchResultsComponent";

describe("search results component", () => {
    const validProps = ["image1", "image2", "image3"]
    const testRenderer = renderer.create(<SearchResults results={validProps}/>);

    it("search results component renders correctly", () => {
        expect(testRenderer).toMatchSnapshot();
    });
});