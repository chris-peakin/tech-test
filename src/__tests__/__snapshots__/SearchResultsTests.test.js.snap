// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search results component search results component renders correctly 1`] = `
Array [
  <p
    className="paragraph"
  >
    Search Results:
  </p>,
  <div
    className="arranged-images"
  >
    <img
      alt="celestial-body"
      className="celestial-body"
      src="image1"
    />
    <img
      alt="celestial-body"
      className="celestial-body"
      src="image2"
    />
    <img
      alt="celestial-body"
      className="celestial-body"
      src="image3"
    />
  </div>,
]
`;
